composer.json

```
@@ -35,7 +35,7 @@
         "doctrine/cache": "^2.0",
         "doctrine/doctrine-bundle": "^2.3",
         "doctrine/orm": "^2.7",
-        "ezyang/htmlpurifier": "^4.10",
+        "ezyang/htmlpurifier": "dev-master as v4.14.0",
```

Euh... No !

---

Well, I know but it looked pretty safe to me: https://github.com/ezyang/htmlpurifier/compare/v4.14.0...master

---

```suggestion
        "ezyang/htmlpurifier": "dev-master#1dd3e52365c32a142fb7c9c9f8f038f18e353270 as v4.14.0",
```

for avoiding another update in master branch